dtale/app.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, *args, **kwargs):
         self.port = kwargs.pop("port", str(random.randint(0, 65535))) or str(
             random.randint(0, 65535)
         )
+        self.port = int(self.port)
```

Comment:
If we're doing the `int` conversion here we may as well convert the line above to no longer wrap the calls to `random.randint` in a `str`

Reply:
I was too eager to fix the problem in test and didn't even see the line above. Also changed the lowest port number to 1025 so testing doesn't need root priv (chances are small though).

Reply:
I'm not able to see these changes when I try to refresh my branch